maps/nadir.dmm

```diff
@@ -19360,6 +19364,11 @@
 	dir = 4
 	},
 /area/station/security/main)
+"iqn" = (
+/turf/space,
+/area/station/hallway/secondary/construction{
+	name = "The Warrens"
+	})
```

Comment:
this is a hole into space, I feel like it shouldn't be here

Reply:
![image](https://github.com/goonstation/goonstation/assets/1017374/4c0614f3-8916-4d68-9a05-9f4ed95f4020)
Leaky station.